server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,9 @@
+import express from "express"
+import { Review } from "../../../models/index.js"
+import showsReviewRouter from "./showsReviewsRouter.js"
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.use("/shows/:id", showsReviewRouter)
```

Comment:
^^ let me know if you have questions about setting up the endpoints in your routers